internal/store/model/device.go

```diff
@@ -93,9 +94,9 @@ func NewDeviceFromApiResource(resource *api.Device) (*Device, error) {
 	}, nil
 }
 
-func (d *Device) ToApiResource() api.Device {
+func (d *Device) ToApiResource(opts ...APIResourceOption) (*api.Device, error) {
```

Comment:
I haven't found a resource / resource list that returns anything other than `nil` here and where you call these methods you also discard the error. Do you foresee this'll change soon?

Reply:
+1